test/fixtures/pubsub_scheduled/main.tf

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google-beta" {
+  project = "${var.project_id}"
+  region  = "${var.region}"
+}
+
+provider "google" {
+  project = "${var.project_id}"
+  region  = "${var.region}"
+}
+
+module "pubsub_scheduled_example" {
+  source = "../../../examples/pubsub_scheduled"
+  project_id = "${var.project_id}"
```

Comment:
`terraform fmt` needs to be executed against the repository.

Reply:
Done